packages/evm/src/precompiles/14-kzg-point-evaluation.ts

```diff
@@ -29,6 +30,17 @@ export async function precompile14(opts: PrecompileInput): Promise<ExecResult> {
     )
   }
 
+  if (opts.gasLimit < gasUsed) {
+    if (opts._debug !== undefined) {
+      opts._debug(`KZG_POINT_EVALUATION (0x14) failed: OOG`)
+    }
+    return OOGResult(opts.gasLimit)
+  }
+
+  if (opts.data.length !== 192) {
```

Comment:
Same here. Can we add tests for this error condition?

Reply:
I didn't want to add those tests, since these are in the Cancun fixtures.

Reply:
But if you want we can add of course.

Reply:
No, that's fine. Will approve

Reply:
Thanks!